lib/validation.js

```diff
@@ -47,26 +48,24 @@ function build (opts) {
       }
     }
 
-    opts[querystringSchema] = ajv.compile(opts.schema.querystring)
+    opts[querystringSchema] = validatorMaker(opts.schema.querystring)
   }
 
   if (opts.schema.params) {
-    opts[paramsSchema] = ajv.compile(opts.schema.params)
+    opts[paramsSchema] = validatorMaker(opts.schema.params)
   }
 }
 
 function validate (handle, params, body, query) {
-  if (handle[paramsSchema] && !handle[paramsSchema](params)) {
-    return inputSchemaError(handle[paramsSchema].errors)
-  }
+  const paramsErrors = handle[paramsSchema] && handle[paramsSchema](params)
```

Comment:
I'll go with `var` here since it is faster :)

Reply:
fixed